WoWPro_Leveling/Wrath/Horde/WOTLK_Howling_Fjord.lua

```diff
@@ -248,14 +251,15 @@ A Give it a Name |QID|12181|N|From Chief Plaguebringer Harris.|
 A Shield Hill |QID|11424|M|53.66,65.27|N|From "Hacksaw" Jenny.|
 A Green Eggs and Whelps |QID|11279|M|53.10,66.90|N|From Plaguebringer Tillinghast.|
 T Give it a Name |QID|12181|N|To Tobias Sarkhoff. Ignore the follow up.|M|52.01,67.39|
-C A Tailor-Made Formula |QID|11305|N|Kill and loot Thorvald.|M|46.77,68.24|
+f New Agamand|M|52.01,67.39|N|Get the flight point at Tobias Sarkhoff.|TAXI|
```

Comment:
@Cagomei says: needs an agument!

Reply:
What should it be? It is for the player to get the flight path.

Reply:
The TAXI tag is just the Flight path name. What TAXI does is if you have that flight path already collected and Cached it will just skip the step.